CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing
+
+We welcome and encourage contributions. We are open minded to suggestions, encourage debate, and aim to build a consensus; if there is no consensus, maintainers will have the final say.
```

Comment:
I read the 'maintainers will have the final say' and felt it was a bit 'strongly worded' for the opening sentence. Could we:
a) make it less so
b) just not write this, as it's essentially covered below in the review/test/release process (maybe make that section more explicit to explain the point)

Reply:
Yes I wasn't sure about putting this in. Perhaps aimed more at internal contributors - I think it's fairly accepted that maintainers have the final say in Open Source. Best to just remove it?

Reply:
👍